### DISCUSSION TITLE
PCA9685 provider for v2?

### DISCUSSION THREAD

**Discussion by caplan** [Pi4J/pi4j#402] [Q&A] [answered]
Does there exist a PCA9685 provider for Pi4J v2?

**Comment by taartspi** [comment, marked as answer]
For that specific device,no.  The pi4j-V2 does not directly provide any devices but does provide example implementations of devices.  If you intend to implement this yourself you could look at the devices in https://www.pi4j.com/examples/. Or https://www.pi4j.com/examples/communityimplementation/.   These contain some mcp230xx devices that hold some similarity to your device and show i2c access of the register sets
